app/renderer/components/preferences/payment/enabledContent.js

```diff
@@ -632,6 +644,23 @@ const styles = StyleSheet.create({
     }
   },
 
+  enabledContent__captcha: {
+    // TODO we need to make sure that text is not in DND zone
```

Comment:
These don't look like they're used here. They're defined in the captcha component as well.

Reply:
fixed